### ISSUE TITLE
[Accessibility] [Keyboard]: Everytime a response button is selected, there is a focus loss and the focus is not visible on any elements in the group 

### ISSUE THREAD

**Issue by rp-dev** [microsoft/BotFramework-WebChat#3106]
## Screenshot
![image](https://user-images.githubusercontent.com/50723361/79495762-f5ff4700-7fd9-11ea-8b8d-41b4c98f32ee.png)

2. Using keyboard tab arrive at "I Accept" button and select it.  Notice that it will show next question group.
3. Arrive at the next group and select "Yes".

## Actual

There is a focus loss.   Actually the focus is on selected button but the focus is not visible. 
Pressing tab takes the focus to next disabled button in the group but the focus is not visible. 
The focus recovers only when user enters the next question group.

## Expected

There should not be focus loss when a button is selected. Also the focus should be clearly visible with every keyboard tab

## WCAG Violation
  https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

**Comment by rp-dev** [comment]
It may be same as https://github.com/microsoft/AdaptiveCards/issues/2236 on adaptive cards but need someone to confirm and comment appropriately.

**Comment by compulim** [comment]
@rp-dev can you send me anything that I can test it on my side (thru Teams)? Either the following will work:

- Bot secret
- URL to the Web Chat page
- Instructions to setup a Health Bot

Then I can take a look and see if it is related to AC 2236 or not.

**Comment by corinagum** [comment]
I'm closing this as stale since we don't have repro steps. If this comes up again, please provide repro steps and we can re-approach.
